package.json

```diff
@@ -3,50 +3,50 @@
   "version": "0.0.1",
   "description": "Converter application (React)",
   "scripts": {
```

Comment:
GLOBAL: Please try to _not change_ any of the workflow tasks / limit changes to the minimum absolutely necessary. Another developer is already in the process of extracting the existing workflows to `builder`, and we want to limit churn there + the existing set is mostly what we'll want to keep targeting.